### DISCUSSION TITLE
Android Support

### DISCUSSION THREAD

**Discussion by ghost** [firecracker-microvm/firecracker#3136] [Ideas]
I'm using an Android VM and Docker versions of Android to test applications.
Is there a plan to support Android?
Currently I'm using Bliss ROM builds

**Comment by xmarcalx** [comment]
Hi @grubeli ,

Sorry but unfortunately at the moment we have no plan to support officially Android VMs. You can have a look to our roadmap https://github.com/firecracker-microvm/firecracker/projects/13. All the main Firecracker use cases are currently centered around Linux VMs.

Nevertheless this is a nice activity that can be carry over by the community. We had in the past members who ported for example FreeBSD VMs running on Firecracker. Feel free to start to work on it and maybe create a new discussion and slack message where you share your experiments, findings and request for help with the community because i am pretty sure other people may find it useful. 

Kind Regards,
Marco

**Comment by seychelles111** [comment]
I guess be happy with what you got
